sphinx/source/docs/user_guide/server.rst

```diff
@@ -762,38 +760,45 @@ thread, e.g. when using ``bokeh.client`` to create sessions from a web
 framework such as Django or Flask. In these cases calling
 ``loop_until_closed()`` from another thread may be an option. Another
 possibility is to not use ``bokeh.client`` at all. See the section
-`userguide_server_embedding`_ for some alternatives.
+:ref:`userguide_server_embedding` for some alternatives.
 
 .. _userguide_server_deployment:
 
 Deployment Scenarios
 --------------------
 
-With an application, we can run it just locally any time we want to interact
-with it. Or we can share it with other people, and they can run it locally
-themselves in the same manner. But we might also want to deploy the application
-in a way that other people can access it. This section describes some of the
-considerations that arise in that case.
+With an application we are developing, we can run it locally any time we want to interact
+with it. To share it with other people who are able to install the required
+python stack, we can share the application with them, and let them run it locally
+themselves in the same manner. However, we might also want to deploy the application
+in a way that other people can access it as a service:
+* without having to install all of the prerequisites
+* without needing to have the source code
+* like any other webpage
+
+This section describes some of the considerations that arise when deploying
+Bokeh server applications as a service for others to use.
```

Comment:
We try to keep a pretty light touch. AFAIK the only current content is an email directing anyone interested to `sales@cio` That said, there is a clear interest in hosted apps, so I am happy to have a *brief* pointer to the new things coming out in AEv5. I'd propose doing that in a separate PR, however, once we can get input from @koverholt and others. It's possible they may want to make a specific landing page for folks directed there fro the Bokeh docs.